### PR TITLE
Add Query to get all applications within tenant

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:
- Create GraphQL Query according to [docs](https://github.com/kyma-incubator/compass/blob/master/docs/architecture/runtime-agent-contract.md#configuring-the-runtime)
- Add ability to fetch all application within tenant

**Related issue(s)**

See #151

### CODE REVIEW

File: components/director/internal/domain/application/repository.go [hunk 1]
```diff
@@ -110,3 +110,23 @@ func (r *inMemoryRepository) findApplicationNameWithinTenant(tenant, name string
 	}
 	return false
 }
+
+// TODO: Make filtering and paging
+func (r *inMemoryRepository) ListByRuntimeID(tenant, runtimeID string, pageSize *int, cursor *string) (*model.ApplicationPage, error) {
+	var items []*model.Application
+	for _, item := range r.store {
+		if item.Tenant == tenant {
+			items = append(items, item)
```
Comment (tgorgol) [inline components/director/internal/domain/application/repository.go]: returned applications should have filtered auths for other runtimes

File: components/director/internal/domain/application/repository.go [hunk 2]
```diff
@@ -110,3 +110,23 @@ func (r *inMemoryRepository) findApplicationNameWithinTenant(tenant, name string
...
+		}
```
Comment (dbadura) [inline components/director/internal/domain/application/repository.go]: returned applications should have filtered auths for other runtimes

File: components/director/internal/domain/root_resolver.go [hunk 3]
```diff
@@ -91,6 +91,10 @@ func (r *queryResolver) Applications(ctx context.Context, filter []*graphql.Labe
 func (r *queryResolver) Application(ctx context.Context, id string) (*graphql.Application, error) {
 	return r.app.Application(ctx, id)
 }
+
```
Comment (tgorgol) [inline components/director/internal/domain/root_resolver.go]:
```suggestion
```

File: components/director/pkg/graphql/schema.graphql [hunk 4]
```diff
@@ -472,6 +472,8 @@ input LabelFilter {
 type Query {
     applications(filter: [LabelFilter!], first: Int = 100, after: PageCursor):  ApplicationPage!
     application(id: ID!): Application
+    applicationsForRuntime(runtimeID: ID!, first: Int = 100, after: PageCursor): ApplicationPage!
+
```
Comment (tgorgol) [inline components/director/pkg/graphql/schema.graphql]:
```suggestion
```

**Pull request merged**